model/registry/messaging.yaml

```diff
@@ -205,4 +205,4 @@ groups:
       - id: system
         type: string
         brief: 'A string identifying the messaging system.'
-        examples: ['kafka', 'rabbitmq', 'rocketmq', 'activemq', 'AmazonSQS']
+        examples: ['kafka', 'rabbitmq', 'rocketmq', 'activemq', 'AmazonSQS', 'gcp_pubsub']
```

Comment:
```suggestion
        examples: ['kafka', 'rabbitmq', 'rocketmq', 'activemq']
```

I'd be inclined to remove all proprietary systems from the examples list. This list doesn't need to be exhaustive.

Reply:
SGTM